packages/taco/test/conditions/base/rpc.test.ts

```diff
@@ -109,7 +143,7 @@ describe('validation', () => {
       expect(result.data).toBeUndefined();
       expect(result.error?.format()).toMatchObject({
         parameters: {
-          _errors: ['Array must contain at least 1 element(s)'],
+          _errors: ['Array must contain at least 2 element(s)', 'Array must contain at least 1 element(s)'],
```

Comment:
I did some digging around to see if there is a way to work around this - it's a little conflicting to have both i.e. `at least 2 element(s)` and `at least 1 element(s)`  - but I couldn't find anything.

Reply:
Yeah, I also found this confusing, but I think this is a side-effect of zod tuples being implemented internally as arrays, where size validation first checks the min and then the max size bound. Not very elegant, but unsure if there's a better way, since we originally wanted to use tuples as each element may have a different type.

Reply:
In any case, it makes sense because there are two possible schemas here, one with two parameters and the other with one, so the first error corresponds to the validation against the two-parameter schema, and the second to the one-parameter schema.